### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by srallen**
Package: N/A

Describe your changes:
For discussion on our Thursday call, I've updated the PR template to include a checklist for what to do post-merge to better align with our deployment process now.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && yarn bootstrap` and app works as expected?
- [ ] Can you run a [production build](https://github.com/zooniverse/front-end-monorepo#getting-started) of the app?

### CODE REVIEW

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -11,11 +13,22 @@ Describe your changes:
...
 - [ ] Is the documentation up to date?
-- [ ] Is the changelog updated?
 
 ## Apps
 
 - [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
 - [ ] Does it work on mobile?
-- [ ] Can you `rm -rf node_modules/ && yarn bootstrap` and app works as expected?
-- [ ] Can you run a [production build](https://github.com/zooniverse/front-end-monorepo#getting-started) of the app?
+- [ ] Can you `yarn panic && yarn bootstrap` and app works as expected?
```
Comment (eatyourgreens) [inline docs/PULL_REQUEST_TEMPLATE.md]: Should we encourage `docker-compose up --build` for testing?

**Comment by rogerhutchings** [comment]
Should we also ask if a storybook has been created if a new component was added?

**Comment by eatyourgreens** [comment]
Should we add stories for all components? I've only been adding them for fairly high level components eg. there's a Recent Subjects story, and a Subject Preview story, but no stories for the individual buttons.

**Comment by rogerhutchings** [comment]
There's probably not much point in adding them for all components, but a reminder to add one if you've added something storybook-worthy couldn't hurt

**Comment by eatyourgreens** [comment]
Definitely something the reviewer should check: does it have a story? Does it have tests?

**Comment by eatyourgreens** [comment]
New components should also be tested for accessibility. At the very least, they should be announced properly in VOiceOver, and should work without a mouse.

**Pull request merged**